ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -172,6 +172,45 @@ message DamlTransactionEntry {
 
   // The time used to derive contract ids
   google.protobuf.Timestamp submission_time = 6;
+
+  // The pre-computed transaction blinding information. Optional.
```

Comment:
Is this optional for now or optional forever?

Reply:
Only for now, starting from the next PR it will always be filled.

Reply:
Cool. If you plan on updating the comment accordingly, I am happy.

Reply:
More precisely, it will be required only when the fetch/lookup nodes in the transactions will be trimmed. But it will be always filled sooner than that.

> If you plan on updating the comment accordingly, I am happy.

Absolutely.